restapi/api/util.go

```diff
@@ -31,15 +31,15 @@ func GetVersion() string {
 func MustMarshal(v interface{}) string {
 	b, err := json.Marshal(v)
 	if err != nil {
-		panic(err)
+		log.WithError(err).Error("Failed marshalling object")
```

Comment:
If it's in a `MustXXX` function then throwing a panic is ok. Also the restapi is not library code, so panic'ing here is usually ok.